lib/seahorn/BvOpSem2.cc

```diff
@@ -553,6 +555,32 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
     setValue(*CS.getInstruction(), res);
   }
 
+  void visitAssertIf(CallSite CS) {
+    Expr ante = lookup(*CS.getArgument(0));
+    Expr conseq = lookup(*CS.getArgument(1));
+    m_ctx.m_z3_solver->reset();
+    m_ctx.m_z3_solver->assertExpr(ante);
+    boost::tribool res = m_ctx.m_z3_solver->solveAssuming(m_ctx.side());
+    if (res == boost::tribool::true_value) {
```

Comment:
the goal of tribool is to hide that it is tribool. Don't use `true_value` directly. Instead:
```
if (res) {  /* true */ }
else if (!res) { /* false */ }
else { /* unknown */ }
```

Reply:
Fixed